tests/pod_builder.rs

```diff
@@ -10,6 +10,27 @@ pub struct PodLifetimeOwner {
 pub struct WasmerciserContainerSpec {
     pub name: &'static str,
     pub args: &'static [&'static str],
+    pub use_private_registry: bool,
```

Comment:
Super Nit: If we are going to use the builder pattern, we should unexport these.